wurst/systems/crafting/CraftingArmory.wurst

```diff
@@ -0,0 +1,66 @@
+package CraftingArmory
+
+// Extension library imports:
+import ItemRestriction
+
+// Local imports:
+import CraftingExtensions
+import ItemRecipeExtended
+import LocalObjectIDs
+import GameMode
+
+
+init
+    GameMode.onModeSelectionFinish() ->
```

Comment:
Same comments as other file pretty much - indent the blocks and use a loop for requirements.